### PR TITLE
post: Tips to succeed at a system’s design interview

### PR DESCRIPTION
**Pull request by luizguilhermefr**

### CODE REVIEW

File: content/posts/2022-02-23-tips-to-succeed-at-a-systems-design-interview/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+### Tip 2: Do not over-engineer
+
+We want to evaluate your ability to create complex systems, but also simple ones.
+In a real-world scenario, you would unlikely start a project with dozens of servers and top-notch tools to intercommunicate them.
+Start lean, paying attention to the scope, and then evolve your solution as variables such as requests per minute are added.
+If you’re in doubt, ask the interviewer.
+
+### Tip 3: Understand what you are proposing
+
+In our industry, we are bombarded with new technologies each day, but if you pay attention, just a few of them really lasts.
```
Comment (gustahrodrigues) [inline content/posts/2022-02-23-tips-to-succeed-at-a-systems-design-interview/index.md:41]: It's a great tip! We're seeing several candidates throwing buzzwords trying to impress the interviewers.

**Review by gustahrodrigues — APPROVED** [review]
Great initiative 🚀 

We could suggest that RH send it to the candidates right after the coding challenge.

**Review by alexantoniomoran — APPROVED** [review]
Well written, thanks for the initiative @luizguilhermefr !

**Review by rcmachado — APPROVED** [review]
That's amazing. I second the suggestion to include it in the interview email we send to the candidates.

Thank you and congrats for the initiative 👏

**Review by vbmendes — APPROVED** [review]

File: content/posts/2022-02-23-tips-to-succeed-at-a-systems-design-interview/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+path: /blog/tips-to-succeed-at-a-systems-design-interview
+date: '2022-02-23'
+comments: true
+---
+
+![cover](./cover.png)
+
+At Loadsmart, the [hiring process](https://github.com/loadsmart/culture/blob/main/hiring/interviewing.md) consists of four steps:
+A screening interview, with one of our recruiters, a coding exercise, done through HackerRank, a system’s design interview, and, finally, a soft-skills interview.
+But what to expect from a system’s design interview, and how to succeed on it?
```
Comment (barbosa) [inline content/posts/2022-02-23-tips-to-succeed-at-a-systems-design-interview/index.md:16]:
```suggestion
But what to expect from a system’s design interview, and how to succeed at it?
```

**Review by barbosa — APPROVED** [review]

**Pull request merged**